lib/package.json

```diff
@@ -28,7 +28,7 @@
     "test-watch": "jest --watch",
     "test-watch-debug": "node --inspect-brk node_modules/.bin/jest --runInBand --watchAll",
     "clean": "rimraf dist junit.xml",
-    "version": "versync -b sync -a"
+    "version": "versync -b sync -a; node ../update-hosted-versions.js"
```

Comment:
Use `npm-run-all`.  Also not sure if invoking node like this is completely safe or not

Reply:
> Use `npm-run-all`. Also not sure if invoking node like this is completely safe or not

This was no longer needed and removed in the current implementation.